src/main.js

```diff
@@ -6,68 +6,96 @@ const components = jsonfile.readFileSync('components.json')
 
 const q = readlineSync.question
 
-let format = q(`Choose a format [${Object.keys(formats["Ro-sham-bo"])}] `)
-if (format === "") {
-  format = "standard"
-}
-// hardcode game and mode
-const game = "Ro-sham-bo"
-let mode = q("Choose a mode (default: leaderboard) ")
-if(mode === "") {
-  mode = "leaderboard"
-}
+// choose game
+let game = q("Choose a game [Ro-sham-bo, Double Ro-sham-bo]: ")
+if (game === "") { game = "Ro-sham-bo" }
+
+// choose format
+let format = q(`Choose a format [${Object.keys(formats)}]: `)
+if (format === "") { format = "standard" }
+
+// choose mode
+let mode = q("Choose a mode [leaderboard, jesseMillerOnly]: ")
+if (mode === "") { mode = "leaderboard" }
 
-const gameComponents = formats[game][format]
+const gameComponents = formats[format]
 
 const getRules = (answer) => {
   return {
     choice: answer,
-    rules: components[game][answer]
+    rules: components[answer]
   }
 }
 
 const findWinner = ([p1Name, p1], [p2Name, p2]) => {
-  let p1Wins = false
-  let p2Wins = false
 
   // Check all the options of which choice wins and loses against the other
   // choice
   if (p1.rules.winsAgainst.includes(p2.choice)) {
-    p1Wins = true
+    return p1Name
```

Comment:
These updates won't quite find a unique winner, they will find the first component that claims to win (or lose) against the other. What this doesn't catch is the case where both components "win against" each other. in which case it should be a draw.

That said, It doesn't really matter for the proof of concept, these game rules work perfectly fine even though they have some edge cases depending on what components people make.